.github/workflows/set-milestone-from-base-branch.yml

```diff
@@ -38,23 +38,24 @@ jobs:
           URL_REGEX='(http(s)?:\/\/.)?(www\.)?[-a-zA-Z0-9@:%._\+~#=]{2,256}\.[a-z]{2,6}\b([-a-zA-Z0-9@:%_\+.~#?&//=]*)'
           UPLIFT_PR_REGEX='uplift\s+(?:of)?:?\s+\K(#\d+)'
           RESOLVE_ISSUE_REGEX="(?:resolv|fix|clos)(?:e|es|ed)?:?\s+\K(${URL_REGEX})"
+          verlte() { printf '%s\n%s' "$1" "$2" | sort -C -V; }
 
-          uplift_pr_body=$(gh pr view "$PR_NUMBER" -R "$REPO_NAME" --json body --jq '.body')
+          uplift_pr_body=$(gh pr view "${PR_NUMBER}" -R "${REPO_NAME}" --json body --jq ".body")
 
           issues=$(echo "${uplift_pr_body}" | grep -ioP "${RESOLVE_ISSUE_REGEX}" || :)
           if [[ -z ${issues} ]]; then
```

Comment:
When the variable is an indexed array, `${variable}` refers to the first element (this isn't a breaking issue on this line, but it's relevant in the rest of the script).
To refer to the whole array, use `${variable[@]}`.
To concatenate the array into a single string, use `${variable[*]}`.

The `${variable[@]}` notation is what you want most of the time. When this is double-quoted, it results in all the elements of the array being quoted individually and remaining separate.